UnitTests/VariablesHelperTests.cs

```diff
@@ -0,0 +1,142 @@
+using NUnit.Framework;
+using static ClassLibrary.VariablesHelper;
+using System;
+
+namespace UnitTests
+{
+    public class VariablesHelperTests
+    {
+        [TestCase(1, 2, 9)]
+        [TestCase(-5, 0, -5)]
+        public void CalculateEquation_WhenNumber1NotEquelNumber2_ShouldCalculateEquation(double a, double b, double expectedResult)
+        {
+            double actualResult = CalculateEquation(a, b);
+
+            Assert.AreEqual(expectedResult, actualResult);
+        }
+
+        [Test]
+        public void CalculateEquation_WhenNumber1EquelNumber2_ShouldThrowException()
+        {
+            try
+            {
+                double actualResult = CalculateEquation(3.5, 3.5);
+            }
+            catch (ArgumentException ex)
+            {
+                Assert.AreEqual("Numbers are same", ex.Message);
+                Assert.Pass();
+            }
+
+            Assert.Fail("No throw exception");
+        }
+
+        [Test]
```

Comment:
Why not test cases but only single test?

Reply:
add additional cases